pkg/keptn/resource_service.go

```diff
@@ -61,10 +88,16 @@ func (r V1ResourceHandler) GetServiceResource(resource string, gitCommitID strin
 	scope.Service(r.Event.Service)
 	scope.Project(r.Event.Project)
 	scope.Stage(r.Event.Stage)
-	scope.Resource(resource)
+	scope.Resource(url.QueryEscape(resource))
+
+	log.Printf("Resource: %s", scope.GetResourcePath())
```

Comment:
I would prefer to not have logging output here, if you really want to preserve this log line it should be more descriptive like `"Fetching Resource from path %s"`